NextcloudTalk/RoomInfoTableViewController.m

```diff
@@ -1816,6 +1817,21 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
                 cell.contactImage.alpha = 1;
                 cell.labelTitle.alpha = 1;
             }
+
+            //Show user status and status message
+            if ([participant.status isEqualToString: kUserStatusDND] && [participant.statusMessage isEqualToString:@""]) {
+                [cell setUserStatus:participant.status];
+                [cell setUSerStatusMessage:NSLocalizedString(@"Do not disturb", nil)withIcon:participant.statusIcon];
+            } else if ([participant.status isEqualToString:kUserStatusAway] && [participant.statusMessage isEqualToString:@""]) {
+                [cell setUserStatus:participant.status];
+                [cell setUSerStatusMessage: NSLocalizedString(@"Away", nil)withIcon:participant.statusIcon];
+            } else if (([participant.status isEqual: kUserStatusDND] && ![participant.statusMessage isEqualToString:@""]) || ([participant.status isEqualToString: kUserStatusAway] && ![participant.statusMessage isEqual:@""])){
+                [cell setUserStatus:participant.status];
+                [cell setUSerStatusMessage:participant.statusMessage withIcon:participant.statusIcon];
+            } else {
+                [cell setUserStatus:participant.status];
+                [cell setUSerStatusMessage:participant.statusMessage withIcon:participant.statusIcon];
+            }
```

Comment:
`[cell setUserStatus:participant.status]` is set in line 1846
so no need to set it for every case in this block.

I would use this if-else block to just set status message